__tests__/spec/data-storage.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-env jest */
+const request = require('supertest')
+const app = require('../../server.js')
+const path = require('path')
+const fs = require('fs')
+const utils = require('../../lib/utils.js')
+
+jest.setTimeout(30000)
+
+function readFile (pathFromRoot) {
+  return fs.readFileSync(path.join(__dirname, '../../' + pathFromRoot), 'utf8')
+}
+/**
+ *
+ */
+describe('The data storage', () => {
+  // check session-data defaults file exists
+
+  it.skip('file should exist', (done) => {
```

Comment:
I'm not too familiar with Jest, but I think you're trying to access the session defaults file from an http request, which won't work. The file exists for the server to set up the default session data, not for access client-side.